engine/renderer/private/gpu_scene.cpp

```diff
@@ -847,12 +853,21 @@ void GPUScene::CreateShadowMapResources()
     shadowSamplerInfo.SetGlobalAddressMode(vk::SamplerAddressMode::eClampToBorder);
     _shadowSampler = _context->Resources()->SamplerResourceManager().Create(shadowSamplerInfo);
 
-    CPUImage shadowCreation {};
-    shadowCreation
+    CPUImage shadowCreationStatic {};
+    shadowCreationStatic
+        .SetFormat(vk::Format::eD32Sfloat)
+        .SetType(ImageType::eShadowMap)
+        .SetSize(2048, 2048)
```

Comment:
would make variables for these so they stay the same size without bugs coming in